docs/source/developers/python.rst

```diff
@@ -177,13 +186,23 @@ Using pip
    to manage your development. Please follow the conda-based development
    instructions instead.
 
-On macOS, install all dependencies through Homebrew that are required for
+On macOS, use Homebrew to install all dependencies required for
 building Arrow C++:
 
 .. code-block:: shell
 
    brew update && brew bundle --file=arrow/python/Brewfile
 
+If you find that your system is missing additional dependencies, consider
+also:
+
+.. code-block:: shell
+
+   brew bundle --file=arrow/c_glib/Brewfile
+
+See `here <./cpp.html#build-dependency-management>`_
```

Comment:
Small sphinx-related feedback: you can reference a location in a different file using `:ref:` (http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations), which gives you a more robust link.

Which in practice means you can do here `` :ref:`here <cpp-development-dependency-management> `` if you add a `.. _cpp-development-dependency-management:` before that title in cpp.rst.

Reply:
Thanks, done.